backend/routes/users.js

```diff
@@ -50,11 +50,27 @@ users.get('/:id', async ctx => {
 });
 
 users.patch('/:id', parsers.json, async (ctx) => {
-    try{ctx.body = await Users.query()
-        .patch({isVerified: true})
-        .findById(ctx.params.id)}
+    let response = {}
+    try{
+        let targetUser = await Users.query()
+        .findById(ctx.params.id)
+
+        //currently, we have to have dateRegistered and lastLogin in patch request or it won't work
+        let patchedPayload = Object.assign(ctx.request.body, {dateRegistered: targetUser.dateRegistered, lastLogin: 0})
```

Comment:
So this means we'd have to send the patch with the whole user object, right? I think we should make it work so we can update just one property. (Meaning get everything from `targetUser`, except what explicitely came from the request)

We have access to lodash at the backend already, so I think let's just do `newUser = _.merge(targetUser, ctx.request.body)`

Reply:
Right now we can patch with just the property we want to update, `lastLogin` and `dateRegistered`, don't need the full user object. Only grabbing `targetUser` so I could patch with the correct `dateRegistered`.